.github/workflows/gofmt.yml

```diff
@@ -0,0 +1,27 @@
+name: gofmt
+
+# We only want one run of this on a branch
+# if you push another commit the previous run should cancel
+concurrency:
+  group: "gofmt-${{ github.ref }}"
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - gh-readonly-queue/main/**
+  pull_request:
+
+jobs:
+  gofmt:
+    name: gofmt
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-go@v3
+      with:
+        go-version: 1.19
+    - uses: actions/checkout@v3
+    - uses: Jerome1337/gofmt-action@v1.0.5
+      with:
+        gofmt-path: './api/golang/ ./container-engine-lib/'
```

Comment:
I'll submit other PRs to fix the other modules and add them here. The ultimate goal is to be able to run it with `./`